ts/Series/Sunburst/SunburstPoint.ts

```diff
@@ -108,27 +108,38 @@ class SunburstPoint extends TreemapPoint {
             moreThanHalf = true;
         }
 
-        if (this.dataLabelPath) {
-            this.dataLabelPath = this.dataLabelPath.destroy();
+        // First time
+        if (!this.dataLabelPath) {
+            // Destroy the path with the label
+            wrap(label, 'destroy', (proceed): undefined => {
```

Comment:
Instead of defining this wrap on every series that supports `textPath` through a custom shape, could we optimize it? It's easy to forget to add this wrap when adding the textPath's support to a new series.

For example, all paths added to `defs` could be by default removed in `label.destroy()`, and all others could be kept (e.g. when the shape is just a point's marker, like in the Networkgraph series).

Reply:
Yes I was in doubt here. The previous behaviour of adding the text path to the label itself in order to be destroyed with it, was a great idea. But doesn't work with the new model. Your proposal is good. Or maybe even safer - use a class name to identify those paths that are tied to a label one-to-one? Or a property in the `SVGElement.textPath` interface. It could simply be `textPath.unique = true`, which would instruct the destructor to flush it down with the label?